CHANGELOG.md

```diff
@@ -5,12 +5,16 @@ however since API is already quite stable we don't expect too much breaking chan
 If we missed a note on some change or you have a questions on migrating from old version,
 feel free to ask us and community.
 
-## Unreleased
+## 0.2.13 (unreleased)
 
 ### Bug Fixes
 
+* fixed signatures of `update`/`insert` methods, some `find*` methods in repositories, entity managers, BaseEntity and QueryBuilders
```

Comment:
What about link to issue after description? I know it is extra work but it is useful for users have a direct link to problem or change.

Reply:
you are right, forgive me lazy 😭